pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/project.json.template

```diff
@@ -0,0 +1,16 @@
+{
+  "dependencies": {
+    "Microsoft.Net.Native.Compiler": "1.6.0",
+    "Microsoft.Private.CoreFx.UAP": "4.4.0-preview1-25205-01",
```

Comment:
that's a good point, I'm not sure how the netcoreapp is pulling WCF in, since I basically tried to mimic what we do over there. cc @ericstj